CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## [v1.0.0](https://github.com/skyway/skyway-js-sdk/releases/tag/v1.0.0) - 2017-09-11
+
+### Added
+
+- Change log
+
+### Fixed
+
+- Change timing of DataConnection 'open' event so that it can always be caught.
```

Comment:
Put a link for PR( #1 ) is easy to find what changed, I think 👀

Reply:
thanks I fixed it!